main.c

```diff
@@ -88,9 +88,11 @@ void segvcatch(int n)
 static const char *
 setfs(char *p)
 {
+#ifdef HISTORIC_FT_WART
 	/* wart: t=>\t */
 	if (p[0] == 't' && p[1] == '\0')
 		return "\t";
+#endif
```

Comment:
This won't compile without HISTORIC_FT_WART defined due to the dangling else.  You need to move the else inside the #ifdef.